rfc3164/message_test.go

```diff
@@ -37,7 +41,7 @@ func (s *Rfc3164MessageTestSuite) TestMessageFromRfc3164(c *C) {
   c.Assert(string(sampleRfc3164Log), Equals, string(*msg.RawMessage()))
   c.Assert("webtest-mark", Equals, msg.Hostname())
   c.Assert("This is a log.info() message", Equals, msg.Message())
-  c.Assert(true, Equals, time.Unix(1402085235, 0).Equal(msg.TimeStamp()))
+  c.Assert(time.Date(2015, 6, 6, 20, 7, 15, 0, time.Now().Location()), Equals, msg.TimeStamp())
```

Comment:
Good looking on the clean up. On an earlier iteration the syslog messages kept a unix timestamp numeric value instead of the TimeStamp type. I bet this is a holder over from that point.